app/controllers/press-releases/overview/planned.js

```diff
@@ -0,0 +1,33 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class PressReleasesOverviewPlannedController extends Controller {  
```

Comment:
Vermijd trailing whitespaces om merge conflicts te vermijden in de toekomst. Kan normaal ingesteld worden in je editor.